app/views/layouts/application.html.erb

```diff
@@ -13,20 +13,42 @@
 
   <body>
     <% unless current_page?(sign_in_path) %>
-      <nav class="crumbs">
-          <ul>
-              <li class="crumb"><a href="#"><%= t(".title") %></a></li>
-              <li class="crumb"><%= link_to t(".sign_in"), sign_in_path %></li>
-          </ul>
-      </nav>
+      <div class="header sticky top-0 bg-white shadow-md flex
+                    items-center px-8 py-02 h-16 flow-root">
+        <div class="float-left mt-3">
+          <a class="p-2 text-3xl" href="#">
+            <%= t(".title") %>
+          </a>
+        </div>
+        <div class="float-right">
+            <nav class="nav font-semibold text-lg">
+              <ul class="flex items-center">
+                <li class="mb-1">
+                </li>
```

Comment:
Do we need this list item?

Reply:
Resolved in this [commit](https://github.com/Esseme/rumblr/pull/24/commits/991d0963268b171e4fe38c1934d5fef424247d29)